### ISSUE TITLE
Visualise stats graphically

### ISSUE THREAD

**Issue by remmycat** [Sendouc/sendou.ink#1048]
The other day someone mentioned in the discord that they have an issue with understanding the new analyzer stats, especially due to their dyslexia. The comparative bar graphs of the spl2 website were much easier to grok for them, than just seeing the numbers next to each other. 

I thought this is important to keep track of and hopefully find some solution in the future :)
Some form of basic graph visualisation is probably helpful for anyone looking at stats, not just people with dyslexia or dyscalculia.

**Comment by Sendouc** [comment]
Thanks for making it into an issue! It definitely would be good to add. Whether it should be behind some toggle or on by default for all depends on what kind of design we could get 🤔
